husarion_ugv_manager/src/lights_manager_node.cpp

```diff
@@ -69,7 +70,7 @@ void LightsManagerNode::Initialize()
     "hardware/e_stop", rclcpp::QoS(rclcpp::KeepLast(1)).transient_local().reliable(),
     std::bind(&LightsManagerNode::EStopCB, this, _1));
 
-  const float timer_freq = this->get_parameter("timer_frequency").as_double();
+  const float timer_freq = this->params_.timer_frequency;
```

Comment:
```suggestion
  const double timer_freq = this->params_.timer_frequency;
```

like in the safety_manager